activerecord-postgis-adapter.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |spec|
   spec.platform = Gem::Platform::RUBY
 
   # ruby-lang.org/en/downloads/branches
-  spec.required_ruby_version = ">= 3.1.0"
+  spec.required_ruby_version = ">= 3.2.0"
```

Comment:
Shouldn't we keep Support for 3.1 until EOL ? (2025-03-31)

Reply:
Rails 8 required ruby_version >= 3.2.0

Reply:
Then, also need to remove ruby 3.1 from the test matrix. 
here: https://github.com/rgeo/activerecord-postgis-adapter/blob/master/.github/workflows/tests.yml#L47C9-L47C36